app/src/release/java/com/futurice/freesound/app/module/InstrumentationModule.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.futurice.freesound.app.module;
+
+import java.util.Collections;
+import java.util.List;
+
+import javax.inject.Singleton;
+
+import dagger.Module;
+import dagger.Provides;
+import okhttp3.Interceptor;
+
+@Module
+public final class InstrumentationModule {
```

Comment:
The both lists of interceptors returned from the release version of the `InstrumentationModule` should always be empty (hence the test). Yes, for now the app interceptors are also empty in the debug version; it's just an expansion point for instrumentation. We don't really care what (or how many) instrumentation interceptors are used by the debug version, but we want to make sure that the release version doesn't any.